client/packages/system/src/Patient/Encounter/ListView.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { FC, useMemo } from 'react';
+import {
+  TableProvider,
+  DataTable,
+  useColumns,
+  createTableStore,
+  NothingHere,
+  createQueryParamsStore,
+  useFormatDateTime,
+  useUrlQueryParams,
+  ColumnAlign,
+} from '@openmsupply-client/common';
+import { EncounterFragment, useEncounter } from './api';
+import { usePatientModalStore } from '../hooks';
+import { PatientModal } from '../DetailView';
+
+type EncounterFragmentWithId = { id: string } & EncounterFragment;
+
+const EncounterListComponent: FC = () => {
+  const {
```

Comment:
this is where the combination of tabs and url query params falls down.
If you go to encounters and sort by something, then go to the programs tab; you get an error shown: `Error: Invalid value for argument "sort.key", enumeration type "ProgramSortFieldInput" does not contain the value "program"
`

the url params will need to be reset to allowable values on the changing to a different tab, or we remove url support for the sorting of the list views which are in tabs

Reply:
yes noticed it as well I created an issue for it: https://github.com/openmsupply/open-msupply/issues/469